pom.xml

```diff
@@ -41,23 +41,13 @@
             <name>Mads Baggesen</name>
             <email>mads.baggesen@eficode.com</email>
         </developer>
-        <developer>
-            <id>gt_dbechtold</id>
-            <name>dbechtold</name>
-            <email>dbechtold@grammatech.com</email>
-        </developer>
-        <developer>
-            <id>aseno-grammatech</id>
-            <name>aseno</name>
-            <email>aseno@grammatech.com</email>
-        </developer>
```

Comment:
developer names should not be removed.  Looks like merge conflicts were not resolved correctly